parsers/MX.py

```diff
@@ -22,6 +31,115 @@
              "BZ->MX-PN": "IntercambioPEN-BEL"
              }
 
+MAPPING = {
+    "Eolica": "wind",
+    "Fotovoltaica": "solar",
+    "Biomasa": "biomass",
+    "Carboelectrica": "coal",
+    "Ciclo Combinado": "gas",
+    "Geotermoelectrica": "geothermal",
+    "Hidroelectrica": "hydro",
+    "Nucleoelectrica": "nuclear",
+    "Combustion Interna": "unknown",
+    "Termica Convencional": "unknown",
```

Comment:
I don't know where to assign these two. Is unknown okay or do you have better ideas?

Reply:
I think unknown is the best we can do at this point